src/components/pages/JudgePage/JudgePage.js

```diff
@@ -278,6 +278,14 @@ export default function JudgePage(props) {
       {judge && (
         <div>
           <div className="imgBox">
+            <h3 className="judgeDisclaimer">
+              {judge.first_name} {judge.middle_initial} {judge.last_name} serves
```

Comment:
On the page to get to this one, there is a period after the middle initial, but not on this one. The period is also on their name in the cases tab, so I'd recommend making this consistent with that.

Reply:
Nice catch! I just made the changes.